tests/mockserver_tests/mock_server_test_base.py

```diff
@@ -78,6 +79,27 @@ def unavailable_status() -> _Status:
     return status
 
 
+# Creates an UNAVAILABLE status with the smallest possible retry delay.
```

Comment:
nit: do we need both this method and the instance method above? If so, could we at least put the main bit of logic into one method?

Reply:
Thanks for catching that, turns out you had already committed that code to main. I've removed it.